tests/run-internalbench.py

```diff
@@ -58,27 +71,70 @@ def run_tests(pyb, test_dict):
 
 
 def main():
-    cmd_parser = argparse.ArgumentParser(description="Run tests for MicroPython.")
-    cmd_parser.add_argument("--pyboard", action="store_true", help="run the tests on the pyboard")
+    cmd_parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="""Run benchmarks for MicroPython.
+
+By default the tests are run against the unix port of MicroPython. To run it
+against something else, use the -t option.  See below for details.
+""",
+        epilog="""\
+The -t option accepts the following for the test instance:
+- unix - use the unix port of MicroPython, specified by the MICROPY_MICROPYTHON
+  environment variable (which defaults to the standard variant of either the unix
+  or windows ports, depending on the host platform)
+- webassembly - use the webassembly port of MicroPython, specified by the
+  MICROPY_MICROPYTHON_MJS environment variable (which defaults to the standard
+  variant of the webassembly port)
+- port:<device> - connect to and use the given serial port device
+- a<n> - connect to and use /dev/ttyACM<n>
+- u<n> - connect to and use /dev/ttyUSB<n>
+- c<n> - connect to and use COM<n>
+- exec:<command> - execute a command and attach to its stdin/stdout
+- execpty:<command> - execute a command and attach to the printed /dev/pts/<n> device
+- <a>.<b>.<c>.<d> - connect to the given IPv4 address
+- anything else specifies a serial port
```

Comment:
Suggest making this string a variable in `run-tests.py` that can be used in that script and here as well, to reduce duplication of this blob of text (and make it easier to update in the future).